configurations/default/server.yml.tmp

```diff
@@ -13,6 +13,15 @@ modules:
     enabled: false
   editor:
     enabled: false
+  deployment:
+    enabled: false
+    ec2:
+      enabled: false
+      subnet: subnet-id
+      securityGroup: security-group-id
```

Comment:
Are we sure that this is information we'd want to include as part of the `appinfo` response?

Reply:
Some of this config info might actually make more sense to store in the `OtpServer` object... that might make more sense than splitting up the AWS info by putting the `targetGroupArn` in the `OtpServer` and this stuff in `server.yml`.

Reply:
Also, there may be more than one EC2 environment we're deploying to, so that just strengthens the argument to place this info into `OtpServer`.

Reply:
I concur.